mod/friends/views/default/resources/friends/of.php

```diff
@@ -6,12 +6,17 @@
  * @subpackage Social.Friends
  */
 
+// needed for correct registration of menu items
+elgg_set_context('friends');
```

Comment:
probably need to pop this context

Reply:
I believe router sets the context from the first route segment anyway. Maybe not even needed